source/_integrations/legrand_homepluscontrol.markdown

```diff
@@ -0,0 +1,103 @@
+
+This `Home+ Control` integration for Home Assistant currently has support for the following devices: 
+- Light switches
+- Power outlets
+
+In both cases, the devices are modelled as simple on/off switches within Home Assistant.
+
+This integration has been tested to work with the following range of Legrand products:
+- Valena Next™ with Netatmo
+
+{% include integrations/config_flow.md %}
+
+## Authentication
+
+Before you are able to configure the Legrand `Home + Control` integration into Home Assistant, you must register with the *Works with Legrand* platform.
+
+These Legrand APIs rely on Oauth2 authentication, so you must follow these steps to obtain the necessary authentication parameters:
+
+1) Register an account at https://developer.legrand.com/
+2) Create a subscription to the *Starter Kit* (currently the only subscription available) and this will generate your SUBSCRIPTION_KEY.
+3) Register an Application, where you will have to define a name, a redirect URL and the scopes of your application (for simplicity you can mark all scopes). 
+
+Once the registered Application is confirmed, you should receive an email containing the CLIENT_IDENTIFIER and the CLIENT_SECRET which you will be using to set up the authentication flows. The Application confirmation email is usually received within a few hours of having issued the request.
+
+### Authentication Flow
+
+Communication with the API, first requires Oauth2 authentication to obtain an access and a refresh token. Subsequent requests to the API, require the use of the SUBSCRIPTION_KEY in addition to the access token.
+
+Information about the Oauth2 exchange is provided here: https://developer.legrand.com/tutorials/0auth2-end-point-url/
+
+As part of the authentication exchange, a callback request is made to the REDIRECT_URL that was entered when you registered the application with the *Works with Legrand* platform. This redirect URL will take on the default value of your Home Assistant instance URL, but because this could differ from the URL that you entered in the Legrand registration form, the integration allows you to explicitly specify the redirect URL parameter that you would like to use.
+
+## Polling Interval Configuration
+
+The Legrand `Home+ Control` API is a cloud-based polling interface. This Home Assistant integration presents 3 different polling intervals:
+1) Plant information request
+2) Plant topology request
+3) Module status request
+
+Refer to the [nomenclature](#api-nomenclature) section for details of what these names mean.
+Refer to the [API limitations](#api-limitations) section to better understand the reason for these polling intervals.
+
+### Plant Information Request
+
+This call retrieves the information of the plants that are associated to a user. It is assumed that this information does not change very often. For the most part, a user will have a single *Legrand Home+ Control* gateway in their home, i.e. a single *plant*, and this will not change frequently.
```

Comment:
```suggestion
This call retrieves the information of the plants that are associated with a user. It is assumed that this information does not change very often. For the most part, a user will have a single *Legrand Home+ Control* gateway in their home, i.e., a single *plant*, and this will not change frequently.
```